Makefile

```
@@ -34,10 +34,8 @@ test:
 	docker-compose -f docker-compose.yml -f docker-compose.test.yml build
 	docker-compose -f docker-compose.yml -f docker-compose.test.yml up --abort-on-container-exit test
 
-# TODO waiting for consensus package management solution
-# update-dependencies:
-#	docker-compose run --rm -T ckan pip --quiet freeze > requirements-freeze.txt
-#	docker-compose run --rm -T ckan pip install -r ${CKAN_HOME}/requirements-freeze.txt
+update-dependencies:
+	docker-compose run --rm -T ckan freeze-requirements.sh `id -u` `id -g` > ckan/requirements.txt
```

```suggestion
	docker-compose run --rm -T ckan freeze-requirements.sh $(shell id -u) $(shell id -g) > ckan/requirements.txt
```

Please test this, my Make skills are :unamused: